common/test/acceptance/accessibility/test_studio_library.py

```diff
@@ -0,0 +1,20 @@
+"""
+Accessibility tests for Studio Library pages.
+"""
+from ..tests.studio.base_studio_test import StudioLibraryTest
+from ..pages.studio.library import LibraryEditPage
+
+
+class StudioLibraryAxsTest(StudioLibraryTest):
+    """
+    Base class to test Studio pages accessibility.
+    """
+
+    def test_lib_edit_page_axs(self):
+        """
+        Check accessibility of LibraryEditPage.
+        """
+        lib_page = LibraryEditPage(self.browser, self.library_key)
+        lib_page.visit()
+        lib_page.wait_until_ready()
+        lib_page._check_for_accessibility_errors()
```

Comment:
Ah, yeah I had thought that we wouldn't use the _check_for_accessibility_errors method in the tests, but instead use the do_axs_audit method and assert on the results. Sorry for not communicating that clearly. I'll put in another commit with what I meant.